R/coord-.r

```diff
@@ -63,28 +63,28 @@ Coord <- ggproto("Coord",
 
   render_fg = function(panel_params, theme) element_render(theme, "panel.border"),
 
-  render_bg = function(panel_params, theme) {
-    abort("Not implemented")
+  render_bg = function(self, panel_params, theme) {
+    cli::cli_abort("{.fn {snake_class(self)}} has not implemented a {.fn render_bg} method")
```

Comment:
Is inlining the function name into the error message needed, or does it now appear in the call?

Reply:
it doesn't with this PR, and I'm still not sure whether it will happen for anything but the layers once I refactor it... so I made the choice to have the other component errors identify the name in the errors